zapisy/apps/users/models.py

```diff
@@ -103,8 +100,12 @@ def consent_granted(self) -> bool:
 
     def get_full_name(self) -> str:
         return self.user.get_full_name()
+
     get_full_name.short_description = 'Użytkownik'
 
+    def is_enrolled_in_native_program(self) -> bool:
```

Comment:
```suggestion
    def is_in_native_program(self) -> bool:
```
Jest nieco zwięźlej, a słowo _enrolled_ ma w SZ inne techniczne znaczenie.